curiouslearning/FeedTheMonsterJS
# Fix: screen get stuck if navigating last page in levelselection (FM-308)

# Changes
- Fix the stuck when navigating to the last page in level selection screen when selecting other language that has fewer levels

# How to test
- In CR, select other language that has fewer levels (e.g. French)
- Open the start screen of FTM.
- Tap or click any part of the start screen to proceed to the next screen
- navigate to the last page and make sure the screen don't stuck

Ref: [FM-308](https://curiouslearning.atlassian.net/browse/FM-308)


[FM-308]: https://curiouslearning.atlassian.net/browse/FM-308?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
